.travis.yml

```diff
@@ -18,6 +18,8 @@ before_script:
 script:
 - make lint vet
 - ./etc/testing/travis.sh
+after_failure:
```

Comment:
This was a bit more idiomatic ... we want this info when tests fail. Was helpful during my debugging.

Reply:
:+1: